core/controllers/profile.py

```diff
@@ -139,6 +153,33 @@ def post(self, _):
 class PreferencesHandler(base.BaseHandler):
     """Provides data for the preferences page."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'PUT': {
+            'update_type': {
+                'schema': {
+                    'type': 'basestring',
+                    'choices': [
```

Comment:
@SD-13 After introducing choices there is no need for the last else clause in the handler -- can you please remove it ?

Reply:
@Nik-09  I have removed that else clause , Please take a look.